mappings/net/minecraft/village/TraderRecipe.mapping

```diff
@@ -5,6 +5,7 @@ CLASS bfj net/minecraft/village/TraderRecipe
 	FIELD d uses I
 	FIELD e maxUses I
 	FIELD f rewardExp Z
+	FIELD g tax I
```

Comment:
I don't understand how this field is used, but I don't think `tax` is correct here

Reply:
It holds a negative int as a discount, but since it is negative, I cannot call it a discount... Hence I called it a "negative" tax.

Reply:
On a side note, this is a flat-rate modification to first slot price after multiplier application; it is not saved to trade itself, but is calculated from a villager based on its rating with the trader and sent to the client with a packet.